advocacy_docs/partner_docs/ChemaxonJChemPostgreSQLCartridge/05-Using.mdx

```diff
@@ -124,15 +119,17 @@ Indexes:
 
 ```
 
-## Searching for data from the Molecule Column
+## Searching for data from the Molecule column
 
-**Substructure Search:**
+You can perform substructure, duplicate, superstructure, and full-fragment searches.
 
-Substructure search finds all structures that contain the query structure as a subgraph. Sometimes the chemical subgraph is not the only thing that is provided, but certain query features that further restrict the structure are also provided as well. If special molecular features are present on the query (eg. stereochemistry, charge, etc.), then only those targets match the ones which also contain the feature. However, if a feature is missing from the query, it is not required to be missing (by default). 
+### Substructure search
 
-Substructure search is performed using the symmetrical sub-/super-structure search operator: `|<|`.
+Substructure search finds all structures that contain the query structure as a subgraph. Sometimes the chemical subgraph isn't the only thing that's provided. Certain query features that further restrict the structure are also provided. If special molecular features are present on the query, such as stereochemistry or charge, then only those targets match the ones that also contain the feature. However, if a feature is missing from the query, it doesn't need to be missing, by default.
 
```

Comment:
Please double check these last two sentences. They were hard for me to understand.